hwtracer/build.rs

```diff
@@ -37,7 +37,7 @@ fn main() {
 
             #[cfg(target_arch = "x86_64")]
             {
-                println!("cargo:rustc-cfg=decoder_ykpt");
+                println!("cargo:rustc-cfg=ykpt");
```

Comment:
This isn't a renaming I'd have done myself, but it's probably not worth arguing about.